scripts/jest/environment.js

```diff
@@ -1 +1,3 @@
+require('core-js/modules/es6.object.assign');
```

Comment:
Just to be clear, this one is writing to the global `Object` but the `Map` one isn't?